tensorboard/webapp/app_routing/dirty_updates_registry_types.ts

```diff
@@ -13,7 +13,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 import {InjectionToken} from '@angular/core';
+import {Selector} from '@ngrx/store';
 
-export const EXPS_WITH_DIRTY_UPDATES_TOKEN = new InjectionToken<string[]>(
-  '[App Routing] Experiments with Dirty Updates'
+import {DirtyUpdates} from './types';
+
+export type DirtyUpdatesSelector<
+  State,
+  Updates extends DirtyUpdates
```

Comment:
Actually, I don't think we would want to let user provide a `Partial<Updates>`. Remove this parameter from generics and require all Selectors to return `DirtyUpdates`.

Reply:
Removed partial.